src/index.ts

```diff
@@ -1,2 +1,54 @@
-// eslint-disable-next-line no-console
-console.log("https://www.youtube.com/watch?v=dQw4w9WgXcQ")
+import path from "path"
+import fs from "fs"
+
+import { unknown } from "./unknown"
+import { optimize } from "./optimize"
+import { Debugger } from "./Debugger"
```

Comment:
in other places, we just console.log - what's the motivation to introduce a debugger class here? I'd roll with the simplest approach first and improve it in separate PRs, but up to you :)

Reply:
`console.log` are always displayed. The debugger can be turned on/off with the config file to see more information. In listings and DH we are printing each file name to the console. I was thinking this might be a bit verbose for most usecases so I removed it with the possibility to configure it

Reply:
I'm ok considering this a separate enhancement. Really up to you

Reply:
Since it's already done, let's see what the others think. It helped me a lot setting up listings, dh and header-footer